### PR TITLE
TE-7819 Replaced Zend usage

### PR DESCRIPTION
**Pull request by romanhavrylko**
- Developer(s): @romanhavrylko

- Ticket: https://spryker.atlassian.net/browse/TE-7666

### CODE REVIEW

**Review by dereuromark — APPROVED** [review]

**Comment by dereuromark** [comment]
Please always check the current version and set the type accordingly.

File: composer.json [hunk 1]
```diff
@@ -16,10 +16,10 @@
   },
   "require": {
     "php": ">=7.2",
+    "laminas/laminas-filter": "^2.0.0",
```
Comment (gechetspr) [inline composer.json]: I would prefer to have `spryker/laminas` here

**Review by gechetspr — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (dereuromark) [reply composer.json]: The hook repo is too internal IMO for this to be needed, no?

Comment (gechetspr) [reply composer.json]: IMO it would be easier to handle all laminas dependencies from the one "source of truth" even for internal things.

Comment (dereuromark) [reply composer.json]: Wouldnt this require more than just this one filter repo?
Either way works for me.

Comment (gechetspr) [reply composer.json]: Yes, but I don't think this is an issue for an internal tool.

I would prefer to have a spryker dependency here

Comment (romanhavrylko) [reply composer.json]: Updated.

**Review by gechetspr — APPROVED** [review]

**Pull request merged**